diff -ruN tempestextremes-2.2.3/mk/system/macosx.make tempestextremes-2.2.3-patch/mk/system/macosx.make
--- tempestextremes-2.2.3/mk/system/macosx.make	2024-07-26 02:04:36.000000000 +0200
+++ tempestextremes-2.2.3-patch/mk/system/macosx.make	2025-01-26 09:13:15.086087903 +0100
@@ -6,13 +6,11 @@
 # Mac OSX (Darwin)
 
 # C++ compiler without and with MPI
-CXX=               g++
-MPICXX=            mpicxx
 
 # Additional C++ command line flags
 
 # NetCDF C library arguments
-NETCDF_ROOT=       /opt/local
+NETCDF_ROOT=       $(PREFIX)
 NETCDF_CXXFLAGS=   -I$(NETCDF_ROOT)/include
 NETCDF_LIBRARIES=  -lnetcdf -lnetcdf_c++
 NETCDF_LDFLAGS=    -L$(NETCDF_ROOT)/lib -Wl,-rpath,$(NETCDF_CXX_ROOT)/lib
